test/built-ins/TypedArray/prototype/indexOf/fromIndex-equal-or-greater-length-returns-minus-one.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-%typedarray%.prototype.indexof
+description: Return -1 if fromIndex >= ArrayLength
+info: >
+  22.2.3.13 %TypedArray%.prototype.indexOf (searchElement [ , fromIndex ] )
+
+  %TypedArray%.prototype.indexOf is a distinct function that implements the same
+  algorithm as Array.prototype.indexOf as defined in 22.1.3.12 except that the
+  this object's [[ArrayLength]] internal slot is accessed in place of performing
+  a [[Get]] of "length".
+
+  22.1.3.12 Array.prototype.indexOf ( searchElement [ , fromIndex ] )
+
+  ...
+  4. Let n be ? ToInteger(fromIndex). (If fromIndex is undefined, this step
+  produces the value 0.)
+  ...
+includes: [testTypedArray.js]
+---*/
+
+testWithTypedArrayConstructors(function(TA) {
+  var sample;
+
+  sample = new TA(42);
+  assert.sameValue(sample.indexOf(0, 42), -1);
+  assert.sameValue(sample.indexOf(0, 43), -1);
+  assert.sameValue(sample.indexOf(0, Infinity), -1);
```

Comment:
I don't think we necessarily need a separate file for `Infinity` (`fromIndex-equal-or-greater-length-returns-minus-one.js`), but if you'd like the assertions separate, you should remove this one from this file.